README.md

```diff
@@ -9,22 +9,60 @@
 
 ## Setup
 
-1. Create an AKS cluster with Azure Dev Spaces by running the following script, specifying the AKS name and region:
+1. Create an AKS resource group, specifying its name and region:
+
+    ```
+    az group create --name bikesharinggroup --location eastus2
+    ```
+
+2. Create a Kubernetes cluster in AKS:
+
+    ```
+    az aks create -g bikesharinggroup -n bikesharingcluster --location eastus2 --kubernetes-version 1.12.5 --node-vm-size Standard_DS2_v2 --node-count 1 --generate-ssh-keys --disable-rbac
+    ```
+
+3. Enable Azure Dev Spaces for the cluster we just created:
+
+    ```
+    az aks use-dev-spaces -g bikesharinggroup -n bikesharingcluster --space dev --yes
+    ```
+
+    We now have an AKS cluster named `bikesharingcluster` in resource group `bikesharinggroup` in the `eastus2` region with Azure Dev Spaces enabled.
+
+4. Select the `dev` parent space:
     
     ```
-    source ./create-aks.sh bikesharing01 eastus2
+    azds space select -n dev
     ```
 
-    This sets up an AKS cluster named `bikesharing01` in resource group `bikesharing01` in the `eastus2` region.
+5. We now need to retrieve the host suffix. To do this, run the following command and note the `HostSuffix` field's value:
 
-2. Run the app's API and Data services.
+    ```
+    azds show-context
+    ```
+
+6. We have to update some of our code with this host suffix. Search for `<REPLACE_ME_BY_HOST_SUFFIX>` strings in the repository, and replace them by the host suffix. Here are the paths of the `values.yaml` files where such replacements should be made:
```

Comment:
Technically, if all you are doing is "helm installing" the parent chart, you only need to replace in ./chart/values.yaml.  The parent chart overrides the child chart values.

Reply:
It doesn't hurt to ask the user to replace everywhere, but I fear it may confuse them especially if they are unfamiliar with Helm.

Reply:
Thanks for your comment. Replaced by "6. We have to update some of our code with this host suffix. Open the `.\chart\values.yaml` file and replace the occurrences of `<REPLACE_ME_WITH_HOST_SUFFIX>` in this file by the host suffix."